### ISSUE TITLE
Fails to activate the microphone in Chrome when used within the Contentful UI

### ISSUE THREAD

**Issue by artasbartas** [nfriedly/contentful-dictate#2]
I managed to get the standalone extension to work in Chrome and Firefox, but when it comes to using it inside the app, it only works inside the Firefox. I suspect that the extension fails to activate the microphone from inside the app. 

Chrome version: 51.0.2704.106 (64-bit)
Console: no errors detected

Any clues on how to fix it?

**Comment by nfriedly** [comment]
Oh, shoot, I switched from including jQuery in the JS bundle to loading it separately, but I made it a http link instead of https. Deploying a new build now.

**Comment by nfriedly** [comment]
Well, that might not be the only issue. It works in chrome outside of the iframe, but apparently not inside of it. I'm looking into it further.

**Comment by artasbartas** [comment]
Yep, I fixed the jquery link locally earlier, but that did not do the trick.

**Comment by nfriedly** [comment]
Figured it out. It needs `allow-same-origin` in the `sandbox` attribute of the `<iframe>` (this must be set by Contentful). See an example here: https://jsfiddle.net/660wvtgL/1/ - in Chrome, only the second one works for me.

**Comment by nfriedly** [comment]
I just had this happen again and noticed that it happened on other sites also. So, I think it's a firefox bug rather than a demo bug.

**Comment by nfriedly** [comment]
Woop, was trying to close a different ticket on a different repo :/
